### PR TITLE
Add feedback on JMS 'Send Message' success & fix TestServiceListener

### PR DESCRIPTION
**Pull request by Matthbo**
Backport of #6517 & poller fix

### CODE REVIEW

File: console/frontend/src/main/frontend/src/app/views/jms/jms-send-message/jms-send-message.component.ts [hunk 1]
```diff
@@ -90,18 +92,22 @@ export class JmsSendMessageComponent implements OnInit {
...
-      this.error = typeof error === 'object' ? error.error : error;
-    }});
+    this.jmsService.postJmsMessage(fd).subscribe({
+      next: (returnData) => {
+        this.error = "";
+        this.processing = false;
+        this.successMessage = 'JMS Message sent successfully';
+      }, error: (errorData: HttpErrorResponse) => {
+        this.processing = false;
+        const error = errorData.error ? errorData.error.error : "An unknown error occured, check the logs for more info.";
```
Comment (philipsens) [inline console/frontend/src/main/frontend/src/app/views/jms/jms-send-message/jms-send-message.component.ts:102]: Didn't the linter pick up the quotes?

**Review by philipsens — COMMENTED** [review]
Code is looking good, but I do have one question.

File: console/frontend/src/main/frontend/src/app/views/jms/jms-send-message/jms-send-message.component.ts (hunk 1 above)
Comment (philipsens) [reply console/frontend/src/main/frontend/src/app/views/jms/jms-send-message/jms-send-message.component.ts:102]: Oh wait, we didn't backport that, right?

Comment (Matthbo) [reply console/frontend/src/main/frontend/src/app/views/jms/jms-send-message/jms-send-message.component.ts:102]: Nope we didn't, sadly it looks a bit messy but hopefully not important anymore soon. I hope that with the 8.1 release we dont need to think about backporting to 8.0 or 7.9 anymore

**Review by philipsens — APPROVED** [review]

**Pull request merged**